src/style.css

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  --header-active-color: #00acdc;
+}
+
+.header {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__item {
+  display: flex;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  position: relative;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  padding: 23px 0;
+}
+
+.nav__link:hover {
+  color: var(--header-active-color);
+}
+
+.nav__link.is-active {
+  color: var(--header-active-color);
+}
+
+.nav__link.is-active:after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  background-color: var(--header-active-color);
+  border-radius: 2px;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+  margin: 10px 0;
+}
+
+.logo__img {
+  display: block;
+  height: 100%;
+  width: 100%;
 }
```

Comment:
Not sure about all those styles
Your `.logo__img` is actually responsible for `<a>` element, which already doesn't make sense. Just set the `height` and `width` directly on <img /> (of course give a class).